common-utils/src/main/java/eu/arrowhead/common/mqtt/filter/ArrowheadMqttFilter.java

```diff
@@ -0,0 +1,10 @@
+package eu.arrowhead.common.mqtt.filter;
+
+import eu.arrowhead.common.mqtt.model.MqttRequestModel;
+
+public abstract class ArrowheadMqttFilter {
```

Comment:
why is this an abstract class if nothing is implemented here? It can be an interface.